From dcdf71aa875961970fc1e5208dcaf13bded6cdbb Mon Sep 17 00:00:00 2001
From: Madalin Bucur <madalin.bucur@freescale.com>
Date: Thu, 24 Jul 2014 20:05:23 +0300
Subject: [PATCH 855/987] dpaa_eth: remove unused code

Removed empty code.

Signed-off-by: Madalin Bucur <madalin.bucur@freescale.com>
Change-Id: I0abc82f3a49b073044d5cabd704ff4771990dcfb
Reviewed-on: http://git.am.freescale.net:8181/15292
Tested-by: Review Code-CDREVIEW <CDREVIEW@freescale.com>
Reviewed-by: Cristian Bercaru <cristian.bercaru@freescale.com>
[Yang: Original patch taken from
QorIQ-SDK-V1.8-SOURCE-20150619-yocto.iso]
Signed-off-by: Quanyang Wang <quanyang.wang@windriver.com>
---
 drivers/net/ethernet/freescale/dpa/dpaa_1588.c | 11 -----------
 1 file changed, 11 deletions(-)

diff --git a/drivers/net/ethernet/freescale/dpa/dpaa_1588.c b/drivers/net/ethernet/freescale/dpa/dpaa_1588.c
index 9f66640..3bf8cbc 100644
--- a/drivers/net/ethernet/freescale/dpa/dpaa_1588.c
+++ b/drivers/net/ethernet/freescale/dpa/dpaa_1588.c
@@ -578,14 +578,3 @@ void dpa_ptp_cleanup(struct dpa_priv_s *priv)
 	kfree(tsu);
 }
 EXPORT_SYMBOL(dpa_ptp_cleanup);
-
-static int __init __cold dpa_ptp_load(void)
-{
-	return 0;
-}
-module_init(dpa_ptp_load);
-
-static void __exit __cold dpa_ptp_unload(void)
-{
-}
-module_exit(dpa_ptp_unload);
-- 
1.9.1

